Readme.md

```diff
@@ -29,13 +29,35 @@ aws_region - AWS region where your Lambda is deployed to
 type = "string"
 required = true
 
-
 aws_service - AWS Service you are trying to access (lambda)
 type = "string"
 required = true
+
+override_target_host - To be used when deploying multiple lambdas on a single Kong service (because lambdas have differennt URLs)
+type = "string"
+required = false
+
+override_target_port - To be used when deploying a lambda on a Kong service that listens on a port other than `443`
+type = "number"
+required = false
+
+override_target_protocol - To be used when deployinglambda on a Kong service that has a protocol different than `https`
+type = "string",
+one_of = "http", "https"
+required = false
 ```
 
-### AWS Setup required
+## Service vs Route scoped configuration
+
+The plugin can be enabled on a per service as well as on a per route basis.
+
+When enabling the plugin on a service/route, the plugin will use the service upstream as the lambda target, unless `override_target_host` is specified.
+This configuration works fine only if a single Lambda is used in the entire service, if multiple lambdas are specified on a per path basis, without the `override_target_host` configured, they will all use the service upstream as the target, which will obviously fail.
```

Comment:
maybe not say that it will "obviously fail",  but explain that having the same host will fail at the lambda level, since it expects a different host.